app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from __future__ import annotations
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.elves.druid import Druid
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+
+
+def calculate_team_total_rating(
+        list_players: list[Druid | ElfRanger]
```

Comment:
are u sure about type annotation for list players here? it seems to me that this function is used for calculating Team rating, it can be not only Elves

Reply:
Thanks.
I didn't know about possibility to use parent class in type annotation